main/modes/mode_tunernome.c

```diff
@@ -747,12 +748,14 @@ void tunernomeMainLoop(int64_t elapsedUs)
                     {
                         // Plot text on top of everything else
                         bool shouldDrawFlat = (semitoneNoteNames[semitoneNum][strlen(semitoneNoteNames[semitoneNum]) - 1] == 1);
-                        int16_t tWidth = textWidth(&tunernome->mm, semitoneNoteNames[semitoneNum]);
+                        char buf[5] = {0};
+                        strncpy(buf, semitoneNoteNames[semitoneNum], 4);
+                        int16_t tWidth = textWidth(&tunernome->mm, buf);
                         if(shouldDrawFlat)
                         {
                             tWidth += tunernome->flatWsg.w + 1;
                         }
-                        int16_t textEnd = drawText(tunernome->disp, &tunernome->mm, c555, semitoneNoteNames[semitoneNum],
+                        int16_t textEnd = drawText(tunernome->disp, &tunernome->mm, c555, buf,
```

Comment:
Why did you copy `semitoneNoteNames[semitoneNum]` into `buf` instead of using it directly like it was before?

Reply:
I wanted to cut off the `\1` at the end, since printing it now causes issues whereas it didn't cause any problems in the past